### DISCUSSION TITLE
无法使用es6方式导入commonjs导出的模块，请问这是umijs的配置缺陷吗？还是说我打开方式不对？

### DISCUSSION THREAD

**Discussion by Lands-1203** [umijs/umi#10918] [Q&A]
![image](https://user-images.githubusercontent.com/61341868/230531515-6c9900c7-b439-4350-95b6-ca7841b4ce73.png)
![image](https://user-images.githubusercontent.com/61341868/230531597-11d1784b-82a8-40a5-ba4c-d0929ea3f0d2.png)
![image](https://user-images.githubusercontent.com/61341868/230531685-62079c31-dbad-4372-abd5-098f4ccf0827.png)

**Comment by Lands-1203** [comment]
我尝试在配置中的chanwebpack使用了webpack插件babel/plugin-transform-modules-commonjs，会报错

**Comment by Lands-1203** [comment]
备注一下：这些包的源码中没有es6lib模块，打包出来的都dist，但是我自己本地测试了使用commonjs导出的变量，也无法使用es6导入

**Comment by fz6m** [comment]
要给最小复现才能看，多半是你的这个包 `package.json` 配的有问题，或者打包这个包的工具有问题，或者你尝试一下 `import * as xxx from 'package'`，然后打印一下 `xxx` 看看有没有方法，这里面涉及到互操作，要看你的这个包构建工具的支持情况。
